experiments/backtest.py

```diff
@@ -62,25 +61,28 @@ def run_backtest(
     post_trade_cash = []
     post_trade_quantities = []
 
-    returns = synthetic_returns(prices).dropna()
-    means = returns.ewm(halflife=125).mean()
-    covariance_df = returns.ewm(halflife=125).cov()
+    returns = prices.pct_change().dropna()
+    means = (
+        synthetic_returns(prices).shift(-1).dropna()
```

Comment:
We need `shift(-1)` here, as when queried at `t`, we want to include the (noisy) returns from `t` to `t+1` (this is also why I needed to shorten the backtest by one day.